content/SCALE/SCALEUIReference/Credentials/LocalGroupsScreens.md

```diff
@@ -9,52 +9,61 @@ tags:
 
 ## Add Group Screen
 
-The **Add User** and **Edit User** configuration screens display the same setting options. 
-Built-in users (except the **root** user) do not include the **Home Directory Permissions** settings, but all new users created, such as those for an SMB share like the **smbguest** user do.
+Click **Add** in the upper right corner of the **Groups** screen to display the **Add Group** configuration screen.
 
-![AddGroupGIDConfigSCALE](/images/SCALE/22.12/AddGroupGIDConfigSCALE.png "Add Group") 
+{{< trueimage src="/images/SCALE/23.10/AddGroupGIDConfigSCALE.png" alt="Add Group Screen" id="3: Add Group Screen" >}}
 
 {{< truetable >}}
 | Setting | Description |
 |---------|-------------|
 | **GID** | Required. Enter a unique number for the group ID (**GID**) TrueNAS uses to identify a Unix group. Enter a number above 1000 for a group with user accounts (you cannot change the GID later). If a system service uses a group, the group ID must match the default port number for the service. |
 | **Name** | Required. Enter a name for the group. The group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), asterisk (*), question mark (?) greater or less than (<) (>), equal (=). You can only use the dollar sign ($) as the last character in a user name. |
-| **Allowed sudo commands** | Enter specific [sudo](https://www.sudo.ws/) commands allowed for this group member, grants administrator permissions for this group member when using these commands. When using sudo, a group member is prompted for their account password. |
-| **Allow all sudo commands** | Select to give this group member administrator permissions and the ability to use all [sudo](https://www.sudo.ws/) commands. When using sudo, a group member is prompted for their account password. | 
-| **Allowed sudo commands with no password** | Enter specific [sudo](https://www.sudo.ws/) commands allowed with no password required for this group member, grants administrator permissions for this group member when using these commands. |
-| **Allow all sudo commands with no password** | Select to give this group member administrator permissions and the ability to use all [sudo](https://www.sudo.ws/) commands with no password required. |
+| **Allowed sudo commands** | Enter the executable path(s) to specific [sudo](https://www.sudo.ws/) commands allowed for members of this group. `/usr/bin/` is the default location for command executables. <br> Grants administrator permissions for group members when using these commands. When using sudo, a user is prompted for their account password. |
+| **Allow all sudo commands** | Select to give group members permission to use all [sudo](https://www.sudo.ws/) commands. When using sudo, a user is prompted for their account password. |
```

Comment:
This can be simplified as well
```suggestion
| **Allow all sudo commands** | Select to give group members permission to use all [sudo](https://www.sudo.ws/) commands. Using sudo prompts the user for their account password. |
```